### PR TITLE
Code cleanup for inject-java

### PR DESCRIPTION
**Pull request by altro3**

### CODE REVIEW

File: inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaEnumConstantElement.java [hunk 1]
```diff
@@ -114,11 +116,6 @@ public boolean isProtected() {
         return false;
     }
 
-    @Override
```
Comment (altro3) [inline inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaEnumConstantElement.java:117]: Unnecessary describe default implementation

File: inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaClassElement.java [hunk 2]
```diff
@@ -633,42 +637,39 @@ public List<ClassElement> getBoundGenericTypes() {
...
     public ClassElement getRawClassElement() {
         return visitorContext.getElementFactory().newClassElement(classElement, elementAnnotationMetadataFactory)
-                .withArrayDimensions(getArrayDimensions());
+            .withArrayDimensions(getArrayDimensions());
     }
 
     @NonNull
     @Override
     public ClassElement withTypeArguments(@NonNull Collection<ClassElement> typeArguments) {
-        if (getTypeArguments().equals(typeArguments)) {
```
Comment (altro3) [inline inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaClassElement.java:659]: This code is incorrect. You try to compare map with collection

File: inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaClassElement.java [hunk 3]
```diff
@@ -618,8 +622,8 @@ public Optional<ClassElement> getEnclosingType() {
             Element enclosingElement = this.classElement.getEnclosingElement();
             if (enclosingElement instanceof TypeElement typeElement) {
                 return Optional.of(visitorContext.getElementFactory().newClassElement(
-                        typeElement,
-                        elementAnnotationMetadataFactory
```
Comment (altro3) [inline inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaClassElement.java:622]: you have 2 styles in these classes. Which `continuation indent` right? 8 spaces or 4 spaces?

File: inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaClassElement.java [hunk 4]
```diff
@@ -431,7 +435,7 @@ public <T extends io.micronaut.inject.ast.Element> List<T> getEnclosedElements(@
      * {@link io.micronaut.inject.ast.ElementFactory#newSourceMethodElement(ClassElement, Object, ElementAnnotationMetadataFactory)}.
      *
      * @param query The query
-     * @param <T>   The element type
```
Comment (altro3) [inline inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaClassElement.java:434]: you have 2 styles in these classes. Do we need `Align parameter / thrown exception decriptions` or not?

File: inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaEnumConstantElement.java [hunk 5]
```diff
@@ -114,11 +116,6 @@ public boolean isProtected() {
...
-    public boolean isPrimitive() {
-        return false;
-    }
```
Comment (altro3) [inline inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaEnumConstantElement.java:120]: Unnecessary describe default implementation

**Review by graemerocher — APPROVED** [review]

File: inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaClassElement.java (hunk 3 above)
Comment (altro3) [reply inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaClassElement.java:622]: @graemerocher could you tell me which codestyle is right?

File: inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaClassElement.java (hunk 4 above)
Comment (altro3) [reply inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaClassElement.java:434]: @graemerocher could you tell me which codestyle is right?

**Comment by altro3** [comment]
@graemerocher Repeat questions:

you have 2 styles in these classes. Which continuation indent right? 8 spaces or 4 spaces?

you have 2 styles in these classes. Do we need Align parameter / thrown exception decriptions or not?

**Comment by graemerocher** [comment]
whatever check style says is correct :)

**Pull request merged**